core/include/kernel/dt.h

```diff
@@ -85,6 +86,18 @@ const struct dt_driver *__dt_driver_end(void);
  */
 int dt_map_dev(const void *fdt, int offs, vaddr_t *base, size_t *size);
 
+/*
+ * Check whether the node at @offs contains TPM Event Log information
+ * (log_addr and log_max_size properties) or not.
+ *
+ * @offs is the offset of the node that describes the device in @fdt.
+ * @buf will contain the phy address of the TPM Event log.
+ * @size will contain the size of the mapped area.
+ *
+ * Returns the size of the mapped area or @DT_INFO_INVALID_TPM_LOG upon failure
+ */
+int dt_get_tpm_log_info (void *fdt, int node, paddr_t *buf, size_t *size);
+
```

Comment:
In my mind, `kernel/dt.h` is meant for DT helper functions potentially used by several callers. This is specific to the TPM log stuff, so I'd rather have this somewhere else.